src/components/TopBarSearch.tsx

```diff
@@ -1,13 +1,28 @@
 import React from 'react';
 import { ReactComponent as Search } from '../assets/search.svg';
-
 import './TopBarSearch.css';
+import { withRouter } from 'react-router-dom';
+
+function TopBarSearch({ history }) {
+    let timer;
+    const id = history.location.pathname.split('/').pop();
+
+    function onChange(e) {
+        const searchValue = e.target.value;
+        clearTimeout(timer);
+        timer = setTimeout((e) => {
+            if(searchValue) {
+                window.location.href = '/block/' + searchValue;
+            }
+        }, 1000);
+    }
```

Comment:
We need a loading spinner so the user knows something is happening (but we can address that in another PR)